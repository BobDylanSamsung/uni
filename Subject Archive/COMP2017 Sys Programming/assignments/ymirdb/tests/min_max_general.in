set a 1 2 3 4 5 6
set b 3 2 5 4 1 7
set c -1 9 3 0 4
set d a b c
set e a 0 b 10 c -2 d
list entries
min a
max a
push a b
get a
min a
max a
push b c
get b
min a
max a
min d
max d
min e
max e
list keys
bye
