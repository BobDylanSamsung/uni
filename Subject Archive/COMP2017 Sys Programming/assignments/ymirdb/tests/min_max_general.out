ok

ok

ok

ok

ok

e [a 0 b 10 c -2 d]
d [a b c]
c [-1 9 3 0 4]
b [3 2 5 4 1 7]
a [1 2 3 4 5 6]

1

6

ok

[b 1 2 3 4 5 6]

1

7

ok

[c 3 2 5 4 1 7]

-1

9

-1

9

-2

10

e
d
c
b
a

bye
